[Bug]: console build error

## Description

[Describe the bug clearly and concisely. What happened vs what you expected?]

**Related PR(s):** #(optional)

**Security considerations:** [If applicable, e.g. auth, env/config exposure]

## Component(s) Affected

- [ ] Core / Backend (app, agents, config, providers, utils, local_models)
- [x] Console (frontend web UI)
- [ ] Channels (DingTalk, Feishu, QQ, Discord, iMessage, etc.)
- [ ] Skills
- [ ] CLI
- [ ] Documentation (website)
- [ ] Tests
- [ ] CI/CD
- [ ] Scripts / Deploy

## Environment

- **CoPaw version:** [e.g. 0.x.x or git commit]
- **OS:** [e.g. macOS 14, Ubuntu 22.04, Windows 11]
- **Install method:** [pip / one-line install / Docker / from source]
- **Python version (if applicable):** [e.g. 3.10]

## Steps to Reproduce

jk_lu@jk-pc:~/Projects/Repos/CoPaw/console$ npm run build

> copaw-console@0.0.0 build
> tsc -b && vite build

src/pages/Control/Channels/components/ChannelCard.tsx:73:15 - error TS2352: Conversion of type 'Record<string, unknown>' to type 'VoiceChannelConfig' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Type 'Record<string, unknown>' is missing the following properties from type 'VoiceChannelConfig': twilio_account_sid, twilio_auth_token, phone_number, phone_number_sid, and 7 more.

73             {(config as VoiceChannelConfig).phone_number ||
                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~

src/pages/Control/Channels/components/ChannelCard.tsx:79:15 - error TS2352: Conversion of type 'Record<string, unknown>' to type 'SingleChannelConfig' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Type 'Record<string, unknown>' is missing the following properties from type 'VoiceChannelConfig': twilio_account_sid, twilio_auth_token, phone_number, phone_number_sid, and 7 more.

79             {(config as SingleChannelConfig).bot_prefix || t("channels.notSet")}
                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 2 errors.


## Actual vs Expected

- **Actual:**
- **Expected:**

## Logs / Screenshots

[Paste relevant log output or attach screenshots. Use code blocks for logs.]

```
(paste logs here)
```

## Additional Notes

[Optional: workarounds, similar issues, etc.]

--- a/console/src/pages/Control/Channels/components/ChannelCard.tsx
+++ b/console/src/pages/Control/Channels/components/ChannelCard.tsx
@@ -1,9 +1,5 @@
 import { Card, Tooltip } from "@agentscope-ai/design";
 import { useTranslation } from "react-i18next";
-import type {
-  SingleChannelConfig,
-  VoiceChannelConfig,
-} from "../../../../api/types";
 import { getChannelLabel, type ChannelKey } from "./constants";
 import styles from "../index.module.less";
 
@@ -28,6 +24,10 @@ export function ChannelCard({
   const enabled = Boolean(config.enabled);
   const isBuiltin = Boolean(config.isBuiltin);
   const label = getChannelLabel(channelKey);
+  const getConfigString = (key: string) =>
+    typeof config[key] === "string" ? config[key] : "";
+  const phoneNumber = getConfigString("phone_number");
+  const botPrefix = getConfigString("bot_prefix");
 
   const getCardClassNames = () => {
     if (isHover) return `${styles.channelCard} ${styles.hover}`;
@@ -69,14 +69,11 @@ export function ChannelCard({
       <div className={styles.cardDescription}>
         {channelKey === "voice" ? (
           <>
-            {t("channels.phoneNumber")}:{" "}
-            {(config as VoiceChannelConfig).phone_number ||
-              t("channels.notSet")}
+            {t("channels.phoneNumber")}: {phoneNumber || t("channels.notSet")}
           </>
         ) : (
           <>
-            {t("channels.botPrefix")}:{" "}
-            {(config as SingleChannelConfig).bot_prefix || t("channels.notSet")}
+            {t("channels.botPrefix")}: {botPrefix || t("channels.notSet")}
           </>
         )}
       </div>

